handler.py

```diff
@@ -0,0 +1,94 @@
+
+import os
+import logging
+import json
+import urllib.parse
+
+from rdflib.plugins.sparql.parser import parseUpdate
+from rdflib.plugins.sparql.algebra import translateUpdate
+
+from sns_client import SnsClient
+from neptune_client import NeptuneClient
+
+
+def main(event, _):
+    # Setup the logger at the INFO level while we continue to profile
+    logger = logging.getLogger()
+    logger.setLevel(logging.INFO)
```

Comment:
Is logging used?

Also, recommend setting up outside method. E.g., at line 13, something like:
```
log = logger.getLogger(__name__)
```

Reply:
I don't think I'm logging in this PR but I am in the follow up. This was in place to debug issues with the AWS Deployment, but not removed as we will want to return the logging for profiling (i.e. how long neptune takes, etc.) I can remove it from this PR for clarity.